### PR TITLE
Pull non-summarized MaxMsgs

### PR DESCRIPTION
**Pull request by mlsteele**
Always pull non-summarized MaxMessages. Seems like the summaries come down as well. This should work as long as at least one of `MaxMsgs` or `MaxMsgsSummaries` comes down.

### CODE REVIEW

File: go/chat/convsource.go [hunk 1]
```diff
@@ -485,28 +495,76 @@ func (m ByMsgID) Len() int           { return len(m) }
...
-func (s *HybridConversationSource) GetMessages(ctx context.Context, convID chat1.ConversationID,
-	uid gregor1.UID, msgIDs []chat1.MessageID, finalizeInfo *chat1.ConversationFinalizeInfo) ([]chat1.MessageUnboxed, error) {
-
-	rmsgsTab := make(map[chat1.MessageID]chat1.MessageUnboxed)
-
-	msgs, err := s.storage.FetchMessages(ctx, convID, uid, msgIDs)
+// getMessagesShared gets messages by ID.
+// The returned messages correspond to `getMsgIDs`.
+// All the new messages in `preFetched` are also saved.
+func (s *HybridConversationSource) getMessagesShared(
```
Comment (mmaxim) [inline go/chat/convsource.go]: I wonder if this function is too complicated, wouldn't it be easier to just reincarnat the old `GetMessagesWithRemotes`? It just seems like this is trying too hard to make one function when two might be better. Let me know what you think.

Comment (mlsteele) [inline go/chat/convsource.go]: Idk, I feel like they're supposed to function exactly the same way and they keep accumulating things like
- handle offline
- sort the messages
- remember to identify the tlf at the end

**Comment by mlsteele** [comment]
Alright, brought back the old one. Added to it sorting and offline error-out.

**Comment by mmaxim** [comment]
CI failed though, I would make sure it wasn't legit, or re-run.

**Pull request merged**